### ISSUE TITLE
Updating for Elm 0.17

### ISSUE THREAD

**Issue by greyarch** [ThomasWeiser/todomvc-elmfire#5]
Any plans to update this for Elm 0.17?

**Comment by ThomasWeiser** [comment]
Yes, I will update as soon as I have [migrated ElmFire](https://github.com/ThomasWeiser/elmfire/issues/13) to Elm 0.17.

**Comment by greyarch** [comment]
Great, looking forward to it.
